thanks-bot/bot.js

```diff
@@ -11,42 +12,46 @@ logger.add(logger.transports.Console, {
 logger.level = 'debug'
 
 // Initialize Discord Bot
-const bot = new Discord.Client({
-  token: auth.token,
-  autorun: true
-})
+const client = new Discord.Client()
 
-bot.on('ready', function (evt) {
+client.on('ready', () => {
   logger.info('Connected')
   logger.info('Logged in as: ')
-  logger.info(bot.username + ' - (' + bot.id + ')')
+  logger.info(`${client.user.username} - (${client.user.id})`)
 })
 
 const ALLOWED_CHANNEL_ID = '525249065039036426'
 
-bot.on('message', function (user, userID, channelID, message, evt) {
-  let thanksMessageRegexp = /!thanks\b/gmi
+client.on('message', message => {
+  const thanksMessageRegexp = /!thanks\b/gmi
 
   // Conditions for exiting the function
```

Comment:
You might rephrase the following comment to be more descriptive:
```suggestion
  // Ignoring the following cases
```